discovery-provider/nginx_conf/nginx.conf

```diff
@@ -41,66 +46,59 @@ http {
 
         location ~* ^/.*_check/?$ {
             proxy_pass http://127.0.0.1:3000;
-            proxy_set_header Host $http_host;
-            proxy_set_header X-Forwarded-For $remote_addr;
         }
 
         location = /version {
             proxy_pass http://127.0.0.1:3000;
-            proxy_set_header Host $http_host;
-            proxy_set_header X-Forwarded-For $remote_addr;
         }
 
         location /v1/metrics {
             proxy_pass http://127.0.0.1:3000;
-            proxy_set_header Host $http_host;
-            proxy_set_header X-Forwarded-For $remote_addr;
         }
 
-        location / {
+        location ~* .*/trending/.* {
```

Comment:
if a user id is passed in, we need to bypass this cache or invalidate far far more frequently (like 500ms).

the problem is if i go repost a song that is on trending and i go view the trending page, that song needs to show as reposted

This request looks like

v1/full/tracks/trending/ePWJD?time=week&limit=8&offset=0&user_id=7eP5n